crd/apis/network/v1/network_types.go

```diff
@@ -12,6 +12,9 @@ const (
 	// name corresponding to the network.
 	// e.g. "networking.gke.io.networks/my-network.IP"
 	NetworkResourceKeyPrefix = "networking.gke.io.networks/"
+	// NodePoolPodRangeLabelPrefix is the prefix for the default Pod range
+	// name for the node and it can be different with cluster Pod range
+	NodePoolPodRangeLabelPrefix = "cloud.google.com/gke-np-default-pod-range"
```

Comment:
Why this label on the network object? This looks like the wrong place for it

Reply:
I think this is a better file for this. crd/apis/network/v1/annotations.go. It has all the other annotations.

Reply:
Then again, this is a label not an annotation.  Let me see if there is a file for node labels.

Reply:
Lets keep it in the annotations file. I don't see  a better place for it. This file isn't the right place because the label is on the node object and this is everything to do with the network object.

Reply:
Sounds good, I include another node label for subnet together in this PR. Hopefully putting them into another const block would be clear.